Dockerfile

```diff
@@ -0,0 +1,21 @@
+############################################################
```

Comment:
By convention the dockerfile is in the base of the project. In this case it is also technically required because the mod_wsgi-dockerfile copies the CWD to the docker container. Res: won't fix.